helpers.js

```diff
@@ -695,15 +702,44 @@ exports.kodiMuteToggle = (request, response) => { // eslint-disable-line no-unus
 };
 
 exports.kodiSetVolume = (request, response) => { // eslint-disable-line no-unused-vars
-    const setVolume = request.query.q.trim();
+    const requestedVolume = request.query.q.trim();
     let Kodi = request.kodi;
+    console.log(`set volume to "${requestedVolume}" percent request received`);
+    return setVolume(Kodi, requestedVolume);
+};
 
-    console.log(`set volume to "${setVolume}" percent request received`);
+const setVolume = (Kodi, volume) => {
+  volume = Math.min(parseInt(volume), 100);
```

Comment:
Let's use 4 spaces. I expected it to be in the lint tbh.

Reply:
Done